bin/run-pipeline.sh

```diff
@@ -6,6 +6,32 @@ FWDIR="$(cd `dirname $0`; pwd)"
 CLASS=$1
 shift
 
+# Set OMP_NUM_THREADS on workers and driver to something appropriate.
+# This is due to OpenBLAS not handling large numbers of cores very well.
```

Comment:
Lets add a link to the keystone issue here. I'll add my openblas commit investigation to the same issue